docs/examples/custom_function/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM icr.io/quantum-public/quantum-serverless-ray-node:0.9.0-py39
```

Comment:
It might be worth creating a template repo in the Qiskit org to store these files so folks can make use of it when writing their own functions. Also would give us a little insight into some of the third party functions out there in the wild